nx-X11/programs/Xserver/hw/nxagent/Keyboard.c

```diff
@@ -75,6 +75,7 @@ is" without express or implied warranty.
 
 #define XKBSRV_NEED_FILE_FUNCS
 #include <nx-X11/extensions/XKBsrv.h>
+#include <nx-X11/extensions/XKBstr.h>
```

Comment:
and why XKBsrv.h here? As I understand,the XKBsrv.h version is libX11 whereas the xkbsrv.h is Xserver. Which one is actually needed here? (I feel, it should be xkbsrv.h here, but that is just a gut feeling).